src/pykeen/models/unimodal/trans_h.py

```diff
@@ -87,27 +88,18 @@ def __init__(
             random_seed=random_seed,
             regularizer=regularizer,
         )
-
         self.scoring_fct_norm = scoring_fct_norm
 
         # embeddings
-        self.normal_vector_embeddings = Embedding.init_with_device(
```

Comment:
seems like this change just made more code

Reply:
here, yes. But it makes the contructors much more readable (in particular for those with two entity and relation embeddings)

Reply:
okay that's a good point too